core/src/main/java/jenkins/telemetry/Telemetry.java

```diff
@@ -151,6 +155,24 @@ public boolean isActivePeriod() {
         return now.isAfter(getStart()) && now.isBefore(getEnd());
     }
 
+    /**
+     * Produces a list of Jenkins core and plugin version numbers
+     * to include in telemetry implementations for which this would be relevant.
+     * @return a map in a format suitable for a value of {@link #createContent}
+     * @since TODO
+     */
+    protected final Map<String, String> buildComponentInformation() {
```

Comment:
```suggestion
    @Restricted(NoExternalUse.class)
    protected final Map<String, String> buildComponentInformation() {
```

Per https://issues.jenkins.io/browse/JENKINS-67332 description to prevent calls from plugins.